recipes/datasets/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "datasets" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 40261f45806ebe003194bb6d14b3f59a6f1e7f9e347e78b662e1ab979ace7e9c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3,<3.7
```

Comment:
Do we know that this won't run on `python >=3.7` and that it's not just that the metadata in the upstream `setup.py` just hasn't been updated? I can see issues posted where people are using python 3.7 and higher and there are no comments that it isn't supported.

Reply:
Testing it